### PR TITLE
Bug/ Chang Example Payload to Open API call

### PR DESCRIPTION
**Pull request by JaneMcBrain**
https://jira.catena-x.net/browse/A1SLDT-258

### CODE REVIEW

File: portal/code/tractus-x-portal/src/components/semantics/SemanticModelDetail.tsx [hunk 1]
```diff
@@ -50,22 +50,23 @@ const SemanticModelDetail = (props) => {
...
-      setExamplePayloadUrl(value);
+      setApiBaseUrl(getOpenApiUrl(id, value))
     }
     
   }
 
-  const onExamplePayloadClick = () => {
-    fetch(examplePayloadUrl)
+  const onOpenApiClick = () => {
+    // needs to be adjusted
```
Comment (JaneMcBrain) [inline portal/code/tractus-x-portal/src/components/semantics/SemanticModelDetail.tsx:64]: @LuLeRoemer should there be a check, if the user added a correct link? If so, I could add an error text, that user input is invalid.

Comment (LuLeRoemer) [reply portal/code/tractus-x-portal/src/components/semantics/SemanticModelDetail.tsx:64]: I guess it makes sense to have some sort of validation (if it is not too complicated) to check whether the URL complies with the documentation: https://swagger.io/docs/specification/api-host-and-base-path/

**Review by LuLeRoemer — COMMENTED** [review]
I would rather not use the OpenAPI docs URL as default for the baseURL

File: portal/code/tractus-x-portal/src/components/semantics/SemanticModelDetail.tsx (hunk 1 above)
Comment (JaneMcBrain) [reply portal/code/tractus-x-portal/src/components/semantics/SemanticModelDetail.tsx]: ok resolved as won't do now.

**Comment by JaneMcBrain** [comment]
@drcgjung I do you know what to do with the failing tests?

**Review by drcgjung — APPROVED** [review]
nearly impossible to review due to lag between feature branch and main.

**Comment by drcgjung** [comment]
@JaneMcBrain  Could you create the final PR against main after a successful integration test, please?

**Pull request merged**